internal/pkg/config/config.go

```diff
@@ -41,6 +41,32 @@ type Config struct {
 	Sources    []sourceMatcher
 	Sinks      []funcMatcher
 	Sanitizers []funcMatcher
+	FieldTags  []fieldTagMatcher
+}
+
+type fieldTagMatcher struct {
+	Key regexp.Regexp
+	Val regexp.Regexp
```

Comment:
These fields do not need to be exported.

Reply:
I think they have to be exported for the JSON Unmarshaling to work :thinking:.

There may be a way around that. If that sounds like something we want to do, I will fix the other matches in another PR (their fields are also exported).

Reply:
Hmmm... That could maybe make sense... Maybe I'm getting my antipatterns backwards, I just thought it seemed strange to export fields of an unexported type.

I suppose of marshalling infers the underlying struct literal, it wouldn't need the type exported for `reflect` to do its job.  But it would need the fields exported in order to set those values.

Weird.  But okay.